test/tst_imageplot.jl

```diff
@@ -0,0 +1,21 @@
+if isinteractive()  # FIXME: Sixel.query_terminal(...) fails when !(stdin isa TTY)
+    img = testimage("monarch_color_256")
+
+    @testset "sixel" begin
+        old_enc = ImageInTerminal.ENCODER_BACKEND[]
+        ImageInTerminal.ENCODER_BACKEND[] = :Sixel
+        p = imageplot(img, title = "sixel")
+        @test p.graphics.sixel[]
+        test_ref("imageplot/img_sixel.txt", @show_col(p, :displaysize => T_SZ))
```

Comment:
It seems that this file is not commited.

Reply:
Xref https://github.com/JuliaIO/Sixel.jl/pull/18#issuecomment-1231525862.

Reply:
But if this file is not commited, how would you expect other people to run the test in interactive mode?

Reply:
Yeah, this is a wip since I get different results when running tests in `mlterm ` or `xterm`.

Reply:
I don't know what to do here, should I remove the tests, since non-interactive mod is bound to fail ?